core/build.gradle

```diff
@@ -26,6 +29,18 @@ android {
     testOptions {
         unitTests {
             includeAndroidResources = true
+            all {
+              useJUnit()
+              if (integrationTests.toBoolean()) {
```

Comment:
Exactly @danielpassos . I am not sure if you have specific use case in mind when you would like to run both unit and integration tests. The only use case I can think of is that you might want to measure combined code coverage. That is something I would like to have a look at soon, but since it can be changed easily I would leave it as it is for now.